gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/IcebergMetadataWriter.java

```diff
@@ -148,6 +158,29 @@
   /* one of the fields in DataFile entry to describe the location URI of a data file with FS Scheme */
   private static final String ICEBERG_FILE_PATH_COLUMN = DataFile.FILE_PATH.name();
 
+  public static final String ICEBERG_COMPLETENESS_ENABLED = "iceberg.completeness.enabled";
+  private static final boolean DEFAULT_ICEBERG_COMPLETENESS = false;
+  private final boolean completenessEnabled;
+  public static final String ICEBERG_COMPLETENESS_WHITELIST = "iceberg.completeness.whitelist";
```

Comment:
Can we refactor the code and move all the config keys for the metadata writer to a separate class?